Apollo-Server/index.js

```diff
@@ -4,22 +4,22 @@ const { ApolloServer, gql } = require("apollo-server");
 // that together define the "shape" of queries that are executed against
 // your data.
 const typeDefs = gql`
-  # Comments in GraphQL strings (such as this one) start with the hash (#) symbol.
-
-  # This "Book" type defines the queryable fields for every book in our data source.
   type Book {
     title: String
     author: String
   }
-
-  # The "Query" type is special: it lists all of the available queries that
-  # clients can execute, along with the return type for each. In this
-  # case, the "books" query returns an array of zero or more Books (defined above).
+  type User {
```

Comment:
create different files for each type of schema and each type of resolver.
You can merge all the schema into one.
Not scalable to have single file having all the resolvers and schema

Reply:
ok Rohith
I'll do that